flappy/main.js

```diff
@@ -378,7 +378,6 @@ function logHighscore(score, duration, vehicle, stage) {
 
     const table = EASY_MODE ? base('Data Explorer Scores') : base('Data Explorer Scores (Hard)');
     table.create({
-        'name': userName,
```

Comment:
I assume this is intentional to remove userName from the table?

Reply:
Yeah, since `userName` is an email